tests/integration/targets/meraki_ms_ospf/tasks/main.yml

```diff
@@ -61,23 +62,24 @@
     check_mode: true
     register: enable_check_mode
 
-  - assert:
+  - name: Assert enable_check_mode
+    ansible.builtin.assert:
       that:
         - enable_check_mode.data is defined
 
   - name: Enable OSPF
-    meraki_ms_ospf:
+    cisco.meraki.meraki_ms_ospf:
       auth_key: '{{ auth_key }}'
-      org_name: '{{test_org_name}}'
+      org_name: '{{ test_org_name }}'
       net_name: IntTestNetworkSwitch
       state: present
       enabled: true
       hello_timer: 20
       dead_timer: 60
       areas:
-        - area_id: 0
-          area_name: Backbone
-          area_type: normal
+        # - area_id: 0
+        #   area_name: Backbone
+        #   area_type: false rmal
```

Comment:
I'd like to understand the reason for commenting out this area and changing it to `false rmal` which I don't think is a valid option.

Reply:
Yeah that is my error with the my changing script. I will do it now.